docs/contribute.md

```diff
@@ -1,111 +1,35 @@
----
----
-# איך לתרום לפרויקט
-
----
-[עמוד הבית](https://linksforisrael.com/)
-[קבוצת הואצאפ שלנו](<https://chat.whatsapp.com/JjD8eijWfDXD10QbM2VyaX>)
 
 ## 🚀 Getting Started
 
-1. **Set Up Your Environment**
-   - Make sure you have Git and Ruby installed.
-
-2. **Fork the Repository**
-   - Go to our [Github Repo](https://github.com/4tal/LinksForIsrael).
-   - Click on the `Fork` button at the top right corner. This will create a copy of the repository in your own GitHub account.
+1. **Prerequisites**
+   - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+   - [Ruby 3](https://www.ruby-lang.org/en/documentation/installation/)
 
-3. **Clone the Forked Repository**
+1. **Clone the Forked Repository**
 
    ```bash
    git clone https://github.com/YOUR_USERNAME/LinksForIsrael.git
    cd LinksForIsrael
    ```
 
-4. **(Optional: If you want to run the GitHub Pages project locally) Install Dependencies**
-   - Install bundler:
-
-     ```bash
-     gem install bundler
-     ```
-
-   - Install project dependencies:
-
-     ```bash
-     bundle install
-     ```
-
-5. **Run Locally**
-
-   ```bash
-   bundle exec jekyll serve
-   ```
-
-   > ⚠️ **Note**: There is currently an issue with running the project locally. The theme doesn't display as expected. If you can solve this issue, your contribution would be greatly appreciated!
+1. **Install Dependencies**
+   - Install bundler: `gem install bundler`
+   - Install project dependencies: `bundle install`
 
-6. **Install Ruby 3**
-   Check how to install an up-to-date version of Ruby for your system [here](https://www.ruby-lang.org/en/documentation/installation/).
+1. ** Setup GitHub API token **
```

Comment:
The spaces nullify the `**`:
<img width="303" alt="image" src="https://github.com/4tal/LinksForIsrael/assets/37291459/1ee17bbe-4307-4484-b3c6-e608e945c145">